database/seeders/GamesTableSeeder.php

```diff
@@ -42,23 +50,147 @@ public function run(): void
         /*
          * add achievements to games
          */
-        Game::all()->each(function (Game $game) {
-            $game->achievements()->saveMany(Achievement::factory()->count(random_int(0, 10))->create([
+        $developers = User::whereHas('roles', function ($q) {
+            $q->whereIn('name', [Role::DEVELOPER, Role::DEVELOPER_JUNIOR, Role::DEVELOPER_RETIRED]);
+        })->pluck('ID')->toArray();
+
+        Game::all()->each(function (Game $game) use ($developers, $faker) {
+            // GameFactory appends a sequence number to every title to ensure uniqueness.
+            // Generate a new title using sequel numbering to ensure uniqueness.
+            $newTitle = ucwords($faker->words(random_int(1, 4), true));
+            if (Game::where('Title', $newTitle)->exists()) {
+                $index = 2;
+                while (true) {
+                    $testTitle = $newTitle . ' ' . match ($index) {
+                        2 => 'II',
+                        3 => 'III',
+                        4 => 'IV',
+                        5 => 'V',
+                        6 => 'VI',
+                        7 => 'VII',
+                        default => strval($index),
+                    };
+
+                    if (!Game::where('Title', $testTitle)->exists()) {
+                        $newTitle = $testTitle;
+                        break;
+                    }
+                    $index++;
+                }
+            }
+            $game->Title = $newTitle;
+            $game->save();
```

Comment:
AchievementFactory always creates a two word name followed by the game ID (i.e "Foo Bar 26"). This attempts to create more natural game names.